### PR TITLE
Add readme and some docs

### PR DESCRIPTION
**Pull request by RuoyuWang-MS**
What does this implement/fix? Explain your changes.
---------------------------------------------------
…

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: azure-container-apps-maven-plugin/QUICKSTART.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Azure Container Apps Maven Plugin Quick Start
```
Comment (allxiao) [inline azure-container-apps-maven-plugin/QUICKSTART.md]: It seems other plugins do not have this `QUICKSTART.md`. Shall we follow the same practice and remove this file?

We can save the content temporarily, and later write a doc in the documentation site. Then, add the link to the Wiki / Quickstarts.

File: azure-container-apps-maven-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Maven Plugin for Azure Container Apps
```
Comment (allxiao) [inline azure-container-apps-maven-plugin/README.md:1]: Might be better to follow the webapp / function's `README.md` structure, given that their plugin is more frequently used and updated.

We can keep the `Azure Container Apps` description section, and add other missed sections.

File: azure-container-apps-maven-plugin/QUICKSTART.md (hunk 1 above)
Comment (RuoyuWang-MS) [reply azure-container-apps-maven-plugin/QUICKSTART.md]: Deleted and add some Wiki pages.

File: azure-container-apps-maven-plugin/README.md (hunk 2 above)
Comment (RuoyuWang-MS) [reply azure-container-apps-maven-plugin/README.md:1]: Function's plugin has the same structure and webapp's has more info. We can fill these part after the detailed doc ready if needed.

**Review by allxiao — APPROVED** [review]

**Pull request merged**